TestResultSummaryService/parsers/BenchmarkParser.js

```diff
@@ -53,45 +27,41 @@ class BenchmarkParser extends Parser {
             let curProductResource = null;
             let isValid = true;
             let curTestData = {};
-
-            // Parse benchmark name
-            if ( ( curRegexResult = benchmarkNameRegex.exec( curItr.value ) ) !== null ) {
-                curBenchmarkName = curRegexResult[1];
-            } else {
-                isValid = false;
-            }
-
-            // Parse benchmark variant
-            if ( ( curRegexResult = benchmarkVariantRegex.exec( curItr.value ) ) !== null ) {
-                curBenchmarkVariant = curRegexResult[1];
+            let varient = "0";
+           
+            if (isNaN(benchmarkVariant)){
```

Comment:
Don't think we need this.

Reply:
Do we have other cases that the value of `benchmarkVariant` is not a number string? if not, I can remove it then.